### PR TITLE
Upgrade the InfluxDB storage-plugin to protocol V3

### PR DESCRIPTION
**Pull request by dmsolr**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [ ] New feature provided
- [ ] Improve performance

- Related issues
#4599
#4615

### CODE REVIEW

**Comment by wu-sheng** [comment]
@kezhenxu94 WDYT about this direction?

**Comment by dmsolr** [comment]
Now, it is ready for review. @mrproliu @wu-sheng @kezhenxu94

File: oap-server/server-storage-plugin/storage-influxdb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/influxdb/TableMetaInfo.java [hunk 1]
```diff
@@ -18,18 +18,80 @@
...
+            ColumnName columnName = column.getColumnName();
+            storageAndColumnMap.put(columnName.getStorageName(), columnName.getName());
+        });
+
+        if (model.getName().endsWith("_traffic")) {
+            // instance_traffic name, service_id
+            // endpoint_traffic name, service_id
+            storageAndTagMap.put(InstanceTraffic.NAME, InfluxConstants.TagName.NAME);
+            if ("instance_traffic".equals(model.getName())
+                || "endpoint_traffic".equals(model.getName())) {
```
Comment (mrproliu) [inline oap-server/server-storage-plugin/storage-influxdb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/influxdb/TableMetaInfo.java:62]: I think we need to use `InstanceTraffic#INDEX_NAME` and `EndpointTraffic#INDEX_NAME` in there.

**Comment by dmsolr** [comment]
@kezhenxu94 , could you help me to see es6 always failed on e2e? I have not modified this partial. (It is also failed on #4666)

**Comment by wu-sheng** [comment]
@kezhenxu94 It seems good now, Need your approve

**Pull request merged**